tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -111,6 +111,9 @@ public static class Png
             // Issue 935: https://github.com/SixLabors/ImageSharp/issues/935
             public const string Issue935 = "Png/issues/Issue_935.png";
 
+            // Issue 1765: https://github.com/SixLabors/ImageSharp/issues/1765
+            public const string Issue1765 = "Png/issues/Issue_1765.png";
```

Comment:
Can we rename this to `Issue1765_Net6DeflateStreamRead` to have a reminder what is the issue about?

Reply:
I don't see why not ;) ill rename somewhere today

Reply:
Renamed the file itself & the constant